resources/test/tests/functional/step_wait.html

```diff
@@ -47,6 +63,12 @@
       "properties": {},
       "status_string": "FAIL"
     }
+    {
+      "name": "async step_wait()",
+      "message": null,
+      "properties": {},
+      "status_string": "PASS"
+    },
```

Comment:
These trailing commas are causing the test failure; this unfortunately has to be actual JSON.

Reply:
Done.